puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman-x-develop-release.groovy

```diff
@@ -17,51 +17,64 @@ pipeline {
     stages {
         stage('Setup Environment') {
             steps {
-                dir('foreman-installer') {
-                    git(url: 'https://github.com/theforeman/foreman-installer.git', branch: 'develop')
-                    script { commit_hash = archive_git_hash() }
+                script {
+                    if ( env.git_ref?.isAllWhitespace() ) {
```

Comment:
is that like Ruby's `&` in `nil&.something`?

Reply:
`?` before a method call makes it nullsafe.